index.src.html

```diff
@@ -877,7 +877,7 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
 interface Report {
   readonly attribute DOMString type;
   readonly attribute DOMString url;
-  readonly attribute ReportBody? body;
+  readonly attribute ReportBody body;
```

Comment:
Why non-nullable? There may be reports that don't have a body — a url and type may be sufficient.